adafruit_shtc3.py

```diff
@@ -123,6 +123,7 @@ def __init__(self, i2c_bus: I2C) -> None:
 
         self._buffer = bytearray(6)
         self.low_power = False
+        self._cached_sleep = False
```

Comment:
I would change this to use the setter so it does the I2C stuff as well:

```suggestion
        self.sleeping = False
```